pkg/hubble/filters/http_test.go

```diff
@@ -371,6 +371,69 @@ func TestHTTPFilters(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		// url filters
+		{
+			name: "url full",
+			args: args{
+				f: []*flowpb.FlowFilter{
+					{
+						HttpUrl:  []string{"https?://cilium.com/docs/[a-z]+", "http://cilium.com/post/\\d+"},
```

Comment:
[CI is complaining](https://app.travis-ci.com/github/cilium/cilium/builds/262783562#L455-L487) about a Go code formatting issue
```suggestion
						HttpUrl:   []string{"https?://cilium.com/docs/[a-z]+", "http://cilium.com/post/\\d+"},
```